### DISCUSSION TITLE
Linkis DataX 引擎安装部署问题

### DISCUSSION THREAD

**Discussion by songer-j** [WeBankFinTech/DataSphereStudio#1127] [Q&A]
1. datax plugin按照部署文档，将部署插件放到{LINKIS_HOME}/linkis/lib/linkis-engineconn-plugins下，并重启服务，调用datax模块的跑数程序，但是报错

查询日志，没有找到hdfswriter/mysqlreader，好像datax整个组件和exchangis-plugins没有关联
![8102b9ff72ec39d749d141ef7f972f7](https://github.com/WeBankFinTech/DataSphereStudio/assets/66104177/6877f801-c303-4c6d-951c-f90beb6fb7a7)
求大佬帮忙解答，datax的整个服务应该放在linkis的哪个位置？？？

2. 安装部署文档的链接路径已改变，建议修改超链接
![image](https://github.com/WeBankFinTech/DataSphereStudio/assets/66104177/51042f2d-c2c8-4d94-9199-267d44c9055b)
3. datax plugin的编译打包路径也改变了，建议修复文档
![image](https://github.com/WeBankFinTech/DataSphereStudio/assets/66104177/031ab77c-15ef-4639-866c-2554690dfc0e)

**Comment by ZqyDaoDao** [comment]
社区不更新了

**Comment by hangpengyue** [comment]
您发的邮件，我已收到。
